bonfire/bonfire.py

```diff
@@ -342,13 +342,20 @@ def _validate_set_image_tag(ctx, param, value):
         raise click.BadParameter("format must be '<image uri>=<tag>'")
 
 
-def _validate_set_prefer(ctx, param, value):
+def _validate_set_preferred_params(ctx, param, value):
     try:
         return split_equals(value)
     except ValueError:
         raise click.BadParameter("format must be '<parameter name>=<value>'")
 
 
+def _validate_set_env_var(ctx, param, value):
```

Comment:
Why have these two functions if they both do the same thing? Couldn't you just have a single `safe_split_equals` function?

Reply:
I debated this myself for a moment too :laughing: It's because I want to have a descriptive error message for each parameter and they each have a slightly different message. I combined them all into one validator func now, take a look.